onnxruntime/core/providers/cuda/tensor/resize_impl.cu

```diff
@@ -10,29 +10,29 @@ using onnxruntime::UpsampleMode;
 
 __device__ int NearestPixel_SIMPLE(float x_original, bool is_down_sampling) {
   if (is_down_sampling) {
-    return static_cast<int>(ceil(x_original));
+    return static_cast<int>(_Ceil(x_original));
   } else {
     return static_cast<int>(x_original);
   }
 }
 
 __device__ int NearestPixel_ROUND_PREFER_FLOOR(float x_original, bool) {
   if (x_original == static_cast<int>(x_original) + 0.5f) {
-    return static_cast<int>(floor(x_original));
+    return static_cast<int>(_Floor(x_original));
   }
-  return static_cast<int>(round(x_original));
+  return static_cast<int>(_Round(x_original));
 }
 
 __device__ int NearestPixel_ROUND_PREFER_CEIL(float x_original, bool) {
-  return static_cast<int>(round(x_original));
```

Comment:
[round()](https://docs.nvidia.com/cuda/cuda-math-api/group__CUDA__MATH__DOUBLE.html#group__CUDA__MATH__DOUBLE_1gbefba28ee84ef32c44d417cfd4f615d4) has different behavior than [rint()](https://docs.nvidia.com/cuda/cuda-math-api/group__CUDA__MATH__DOUBLE.html#group__CUDA__MATH__DOUBLE_1g3b8026edb2f2e441669845f0f3fa3bf7) (which _Round() uses) for halfway values.
is that ok?

Reply:
@duli2012 Any comments?

Reply:
does it work with roundf()? the argument is a float.

Reply:
round_perfer_ceil means round half up, so round() should be right here?